web/public/index.html

```diff
@@ -32,26 +29,31 @@
         var curlyBracket = String.fromCharCode(123);
         var startingTemplateToken = `${curlyBracket}${curlyBracket}`;
         if (!value || value.startsWith(startingTemplateToken)) {
-          return defaultValue
+          return defaultValue;
         }
 
-        return value
+        return value;
       }
 
       function injectVariables() {
-
         window.ENV = {
-          measurementId: getTemplateValue("{{ .MeasurementId }}", ""),
-          analyticsEnabled: getTemplateValue("{{ .AnalyticsEnabled }}", false),
-          serverPathPrefix: getTemplateValue("{{ .ServerPathPrefix }}", "/")
+          measurementId: getTemplateValue('{{ .MeasurementId }}', ''),
+          analyticsEnabled: getTemplateValue('{{ .AnalyticsEnabled }}', 'false'),
+          serverPathPrefix: getTemplateValue('{{ .ServerPathPrefix }}', '/'),
         };
 
         var base = document.createElement('base');
-        base.href = window.ENV.serverPathPrefix
+        base.href = window.ENV.serverPathPrefix;
         document.getElementsByTagName('head')[0].appendChild(base);
-      };
+      }
 
       injectVariables();
+      
+      if (window.ENV.analyticsEnabled === 'true') {
```

Comment:
We are using the same keys as we did for the backend

Reply:
And we'll avoid loading the script if analytics are disabled

Reply:
is it ok to have the `_writeKey` hardcoded?

Reply:
It will come from the backend at some point. So for the moment, it will be disabled. But yes someone could open the console and look at it and that's fine